addons/sale/tests/test_sale_to_invoice.py

```diff
@@ -414,78 +414,6 @@ def test_multi_company_invoice(self):
         downpayment.create_invoices()
         self.assertEqual(so_for_downpayment.invoice_ids[0].company_id.id, so_company_id, "The company of the downpayment invoice should be the same as the one from the SO")
 
-    def test_invoice_analytic_account_default(self):
```

Comment:
These tests are still interesting, we don't want to have the default distribution on the invoice if we had a custom one on the SO